packages/gcc7.rb

```diff
@@ -70,7 +70,6 @@ def self.build
 
   def self.check
     Dir.chdir('build') do
-      system 'ulimit -s 32768'
```

Comment:
You couldn't have just told me that `ulimit` is already set at unlimited by default?